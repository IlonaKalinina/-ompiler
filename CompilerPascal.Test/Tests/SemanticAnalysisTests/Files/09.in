program Test_09;
var
a : integer;
begin
a := 1 + 1.5;
end.