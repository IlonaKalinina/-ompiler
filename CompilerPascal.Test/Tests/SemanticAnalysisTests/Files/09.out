(5,8) ERROR: Object reference not set to an instance of an object.
